source/Modules/Devon4Net.Infrastructure.AWS.CDK/Stack/ResourceStacks/ECSProvisionStack.cs

```diff
@@ -166,7 +166,7 @@ private void CreateEc2Service(EcsServiceOptions service, EcsTaskDefinitionOption
                 throw new ArgumentException("Please add a task definition option properly set up on your json configuration. No task definition could be added.");
             }
 
-            var cluster = StackResources.EcsClusters.FirstOrDefault(c => c.Key == service.EcsClusterId).Value;
+            var cluster = StackResources.EcsClusters.FirstOrDefault(c => c.Key == service.EcsClusterId).Value as Cluster;
```

Comment:
Please consider using "pattern maching" statement:
            if (StackResources.EcsClusters.FirstOrDefault(c => c.Key == service.EcsClusterId).Value is not Cluster cluster)
            {
                throw new ArgumentException("Please add a cluster definition option properly set up on your json configuration. No cluster could be added.");
            }